File: code/backend/python/app/core/models/story/story_import_result.py
from dataclasses import dataclass

@dataclass
class StoryImportResult:
    storyUuid: str
    status: str
    textsImported: int = 0
    locationsImported: int = 0
    eventsImported: int = 0
    itemsImported: int = 0
    difficultiesImported: int = 0
    classesImported: int = 0
    choicesImported: int = 0

File: code/backend/php/src/Core/Domain/Story/StoryImportResult.php
<?php

declare(strict_types=1);

namespace Games\Paths\Core\Domain\Story;

class StoryImportResult
{
    public function __construct(
        public string $storyUuid,
        public string $status,
        public int $textsImported = 0,
        public int $locationsImported = 0,
        public int $eventsImported = 0,
        public int $itemsImported = 0,
        public int $difficultiesImported = 0,
        public int $classesImported = 0,
        public int $choicesImported = 0
    ) {
    }
}
